dags/multipod/maxtext_nightly.py

```diff
@@ -28,7 +28,7 @@
 with models.DAG(
     dag_id="maxtext_nightly",
```

Comment:
How are we logically dividing the DAGs, given the end-to-end tests run using `nightly` as well? Maybe this DAG's ID is a misnomer and it should be `maxtext_nightly_step_time`? cc @tonyjohnchen in case you have any thoughts on this.

Reply:
I used `maxtext_nightly` at first since it should run maxtext with libtpu-nightly every day. I am open to change the name since we will add more tests for maxtext with libtpu-nightly